templates/sshd_config.j2

```diff
@@ -1,5 +1,9 @@
 {{ ansible_managed | comment }}
 
+{% if ssh_version.stdout is version('8.2', '>') %}
+Include /etc/ssh/sshd_config.d/*.conf
+{% endif %}
+
```

Comment:
Given previous recommendations, this block can be removed, as it will be templated directly from the configuration.